ua-snap/iem-webapp
# Switch app to fetch places from the API instead of local copies

Closes #200 

To test:

 * Open the app and try selecting a community, then a HUC, then a protected area.  Make sure the report loads for each of these types.
 * In a fresh tab, open the app directly to http://localhost:3000/data  (not the home page), then navigate to the home page and test that searching for locations works.  (This tests a condition where the app is loaded, but won't yet have tried to fetch the list of places.)

Some things aren't accounted for here:

 * Simulate super-slow network.  What happens?
 * What happens if the network request to load the places fails entirely?  What should happen?
